olive/data/component/dataloader.py

```diff
@@ -48,3 +55,120 @@ def rewind(self):
             self.data_iter = None
 
     return _CalibrationDataReader(dataloader, **kwargs)
+
+
+@Registry.register_dataloader()
+def default_snpe_dataloader(_dataset, snpe_model, batch_size=None):
+    from olive.snpe.data_loader import SNPEProcessedDataLoader
+    from olive.snpe.utils.input_list import create_input_list
+
+    class _SNPEProcessedDataLoader(SNPEProcessedDataLoader):
+        def __init__(self, dataset, snpe_model, batch_size=None):
```

Comment:
>snpe_model [](http://example.com/codeflow?start=36&length=10)

seems we only need the io_config?

Reply:
yes, it only needs the `input_names` and `input_shapes`.

Reply:
if so, should we pass io_config here instead of the mdoel itself?

Reply:
Done.